plugins/dell-esrt/README.md

```diff
@@ -0,0 +1,38 @@
+Dell ESRT Support
+=================
+
+Introduction
+------------
+
+This allows installing Dell capsules that are not part of the ESRT table.
+
+Build Requirements
+------------------
+
+For Dell support you will need libsmbios_c version 2.3.0 or later.
+
+* source:		http://linux.dell.com/cgi-bin/cgit.cgi/libsmbios.git/
```

Comment:
Ah this hasn't been updated in a while.  this should actually be https://github.com/dell/libsmbios
And we can just tell people to grab RPMs from upstream releases too.
https://github.com/dell/libsmbios/releases

Reply:
Makes sense; can you fix up both READMEs pls.

Reply:
fixed